Shared/Views/ShareCampaignView.swift

```diff
@@ -42,17 +42,23 @@ struct ShareCampaignView: View {
     
     var entryView: some View {
         EntryView(campaign: $widgetData, showMilestones: showMilestones, preferFutureMilestones: preferFutureMilestones, showFullCurrencySymbol: showFullCurrencySymbol, showGoalPercentage: showMainGoalPercentage, showMilestonePercentage: showMilestonePercentage, appearance: appearance, useNormalBackgroundOniOS17: true)
-            .frame(minWidth: 350, maxWidth: 350, minHeight: 200, maxHeight: 350)
+            .frame(minWidth: 350, maxWidth: 350, minHeight: 200, maxHeight: 450)
             .clipShape(RoundedRectangle(cornerRadius: (clipCorners ? 15 : 0)))
     }
     
     @ViewBuilder
+    @MainActor
```

Comment:
Do we want to add a comment to remind future us that we do this for the ImageRenderer?